pkg/azure/api/api.go

```diff
@@ -406,14 +406,20 @@ func (c *Client) AssignPrivateIpAddressesVMSS(ctx context.Context, instanceID, v
 		return fmt.Errorf("interface %s does not exist in VM %s", interfaceName, instanceID)
 	}
 
```

Comment:
Could you document this particular API limitation in the code here?

Reply:
Fixed.